modules/gdscript/gdscript_parser.cpp

```diff
@@ -3135,10 +3135,9 @@ void GDScriptParser::get_class_doc_comment(int p_line, String &p_brief, String &
 		String striped_line = doc_line.strip_edges();
 
 		// Set the read mode.
-		if (striped_line.begins_with("@desc:") && p_desc == "") {
+		if (striped_line.is_empty() && mode == BRIEF) {
```

Comment:
Is the `!p_brief.is_empty()` check necessary here? Should we allow blank lines like:

```gdscript
##
## Brief.
##
## Description.
##
```

Or we will only allow:

```gdscript
## Brief.
##
## Description.
```

Reply:
I personally wouldn't allow extraneous lines at the top and bottom, but that's just me.